packages/smooth_app/lib/pages/scan/ml_kit_scan_page.dart

```diff
@@ -102,28 +102,33 @@ class MLKitScannerPageState extends State<MLKitScannerPage> {
 
     return LayoutBuilder(
       builder: (BuildContext context, BoxConstraints constraints) {
-        return Stack(
-          fit: StackFit.expand,
-          children: <Widget>[
-            SmoothRevealAnimation(
-              delay: 400,
-              startOffset: Offset.zero,
-              animationCurve: Curves.easeInOutBack,
-              child: Transform.scale(
-                scale: scale,
-                child: Center(
-                  child: CameraPreview(
-                    _controller!,
-                  ),
+        final List<Widget> children = getScannerWidgets(
+          context,
+          constraints,
+          _model,
+        );
+
+        //Inserting the scanner at the right position
+        children.insert(
+          1,
+          SmoothRevealAnimation(
```

Comment:
Same remark: that `1` stinks. Luckily it's the same `1` in both cases: please add a Widget parameter to `getScannerWidgets` and put it as item `1`.

Reply:
Already working on it 👍🏼